infer-web.py

```diff
@@ -1142,7 +1142,7 @@ def change_f0_method(f0method8):
                 )
                 with gr.Row():
                     trainset_dir4 = gr.Textbox(
-                        label=i18n("输入训练文件夹路径"), value="E:\\语音音频+标注\\米津玄师\\src"
+                        label=i18n("输入训练文件夹路径"), value=i18n("E:\\语音音频+标注\\米津玄师\\src")
```

Comment:
Well, this file is just a demo path so maybe no need to translate it.

Reply:
Probably true, and feel free to ignore it.  I did note that all of the other paths listed on the web UI were translated, so my goal was to make a english user only see english throughout the entire UI.  Understood if that's not wanted.

Reply:
All right.